python/cuxfilter/tests/charts/datashader/test_graph_assets.py

```diff
@@ -50,15 +50,4 @@ def test_calc_connected_edges(
             node_y_dtype=cp.float32,
         ).reset_index(drop=True)
 
-        res = (
-            res.compute().reset_index(drop=True)
-            if isinstance(res, dask_cudf.DataFrame)
-            else res
-        )
-        result = (
-            result.compute()
-            if isinstance(result, dask_cudf.DataFrame)
-            else result
-        )
-
-        assert res.to_pandas().equals(result.to_pandas())
+        assert_eq(res, result, check_divisions=False, check_index=False)
```

Comment:
We can only use `check_index=False` if the ordering of rows is not important. Not sure if this is the case for `calc_connected_edges`, but I have a feeling it might be.

Reply:
Any thoughts @AjayThorve ? Seems like this PR resolves query-planning failures if this test can be modified. Otherwise, I'll need to investigate the differences in row ordering further.

Reply:
You are right, the ordering is not important for calc_connected_edges, so it's safe to use check_index=False.